nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/resources/OSGI-INF/comment-pageprovider-contrib.xml

```diff
@@ -15,6 +15,7 @@
       <pattern>
         SELECT * FROM Comment WHERE comment:parentId = ?
       </pattern>
+      <sort column="comment:creationDate" ascending="false"/>
```

Comment:
To double-check, the service has
```
        GET_COMMENTS_FOR_DOC_PAGEPROVIDER_NAME, singletonList(new SortInfo("dc:created", true)), pageSize,
                currentPageIndex, props, documentId);
```
so this must be ineffective. What's the difference between `dc:created` and `comment:creationDate` btw?

Reply:
That singletonList was removed in order to follow the contrib.

Your question is very relevant, since, currently, we need to pass the comment:creationDate as an argument and it is not validated. This means that we can create comments with past dates or, even, in the future. I raised this question for Team Core, but still without any reply.

Anyway, since we are relying on comment schema fields to represent comments, the usage of comment:creationDate as sort column was only for coherence purposes.

Reply:
I don't remember seeing the question about `comment:creationDate`. Could you re-ask in #team-core please?

Reply:
> I don't remember seeing the question about `comment:creationDate`. Could you re-ask in #team-core please?

Done!

Reply:
> To double-check, the service has
> 
> ```
>         GET_COMMENTS_FOR_DOC_PAGEPROVIDER_NAME, singletonList(new SortInfo("dc:created", true)), pageSize,
>                 currentPageIndex, props, documentId);
> ```
> so this must be ineffective. What's the difference between `dc:created` and `comment:creationDate` btw?

This is legacy from former service.